app/javascript/components/recipes/edit/recipe-edit.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <div class="flex flex-col">
+    <!-- nombre -->
+    <div class="flex items-center">
+      <a :href="`/recipes/${recipeId}`">
+        <img
+          class="h-6 w-6 text-white mr-2"
+          svg-inline
+          src="../../../../assets/images/arrow-left-svg.svg"
+        >
+      </a>
+      <div class="h-7 font-sans font-lg text-2xl text-black font-bold flex-grow">
+        {{ recipe.name }}
+      </div>
+    </div>
+    <!-- cuadro blanco edición -->
+    <div class="flex flex-col py-8 px-6 w-auto h-auto bg-gray-50 flex-grow-0 my-4">
+      <!-- datos básicos -->
+      <div
+        class="h-7 w-auto font-hind font-bold text-lg text-black flex-none self-stretch flex-grow-0 mb-8"
+      >
+        1. {{ $t('msg.recipes.basic') }}
+      </div>
+      <div class="flex flex-col">
+        <div class="flex items-start justify-between h-16 flex-none self-stretch flex-grow-0 mb-8">
+          <div class="relative w-2/5">
+            <div class="text-gray-600 text-sm absolute bg-gray-50 px-1 left-2 -top-2">
+              {{ $t('msg.recipes.name') }}
+            </div>
+            <input
+              class="w-full h-16 bg-gray-50 border border-gray-600 box-border rounded-md flex-none flex-grow-0 px-5"
+              v-model="recipe.name"
+            >
+          </div>
+          <div class="relative w-1/4">
+            <div class="text-gray-600 text-sm absolute bg-gray-50 px-1 left-2 -top-2">
+              {{ $t('msg.recipes.portions') }}
+            </div>
+            <input
+              class="w-full h-16 bg-gray-50 border border-gray-600 box-border rounded-md flex-none flex-grow-0 px-5"
+              v-model="recipe.portions"
+            >
+          </div>
+          <div class="relative w-1/4">
+            <div class="text-gray-600 text-sm absolute bg-gray-50 px-1 left-2 -top-2">
+              {{ $t('msg.recipes.preparation') }}
+            </div>
+            <input
+              class="w-full h-16 bg-gray-50 border border-gray-600 box-border rounded-md flex-none flex-grow-0 px-5"
+              v-model="recipe.cookMinutes"
+            >
+          </div>
+        </div>
+      </div>
+      <!-- ingredients -->
+      <div class="h-7 w-auto font-hind font-bold text-lg text-black flex-none self-stretch flex-grow-0 mb-8">
+        2. {{ $t('msg.recipes.ingredients') }}
+      </div>
+      <recipe-ingredients
+        :recipe-ingredients="recipe.recipeIngredients.data"
+        @add-ingredient="addIngredient"
+        @delete-ingredient="deleteIngredient"
+        @increase-quantity="increaseQuantity"
+        @decrease-quantity="decreaseQuantity"
+      />
+      <!-- pasos -->
+      <recipe-steps
+        :recipe-steps="recipe.steps.data"
+        @new-step="addStep"
+        @delete-step="deleteStep"
+      />
+      <!--  botones -->
+      <div class="flex items-start w-auto h-11 flex-none self-stretch flex-grow-0">
+        <button 
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing spaces not allowed  no-trailing-spaces